reactive/etcd.py

```diff
@@ -180,10 +180,12 @@ def configure_etcd():
                           cluster_data, owner='root', group='root')
     else:
         # render systemd
-        templating.render('systemd', 'files/systemd/etcd.service',
+        templating.render('systemd', '/etc/systemd/system/etcd.service',
                           cluster_data, owner='root', group='root')  # noqa
```

Comment:
This looks wrong to me, because on Xenial there should already be an etcd.service file installed by the package (in /usr/lib/systemd/system), and the 'systemctl enable etcd' call just below will symlink that into /etc/systemd/system.  So the charm shouldn't need to generate an independent copy of the service file.